src/app/app.component.ts

```diff
@@ -20,6 +25,8 @@ export class AppComponent {
 
   initializeApp() {
     this.platform.ready().then(() => {
+      this.store$.dispatch(LoadAppVersion());
+      this.store$.dispatch(LoadEmployeeName());
```

Comment:
It's a bit too early for dispatching `LoadEmployeeName` as the user is not logged in just yet. Will always be unknown at this point.

A good place to dispatch this action is after the user is authenticated and the token is acquired.